jib-maven-plugin/src/test/resources/maven/projects/spring-boot-multi/service-1/pom.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>service-1</artifactId>
+  <version>1.0.0.TEST-SNAPSHOT</version>
+
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>2.1.6.RELEASE</version>
+  </parent>
+
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>com.google.cloud.tools</groupId>
+          <artifactId>jib-maven-plugin</artifactId>
+          <version>@@PluginVersion@@</version>
+        </plugin>
+      </plugins>
+    </pluginManagement>
+  </build>
```

Comment:
I bet you've copied this whole setup from another directory, but isn't this redundant?

Reply:
is this maybe an indication that we might need to do some test cleanup?

Reply:
oops, just found https://github.com/GoogleContainerTools/jib/issues/2150 again.

Reply:
Ah, then I think the `pluginManagement` in the root `pom.xml` is unnecessary?

Reply:
I think I want to keep it to ensure that the output isn't printed on the root project, even though the Jib plugin is applied?